pkg/component/watch_test.go

```diff
@@ -16,41 +16,50 @@ import (
 func TestIsRegExpMatch(t *testing.T) {
 
 	tests := []struct {
-		testName   string
-		strToMatch string
-		regExps    []string
-		want       bool
-		wantErr    bool
+		testName      string
+		directoryName string
+		strToMatch    string
+		regExps       []string
+		want          bool
+		wantErr       bool
```

Comment:
unit test is not covering the `return false, err` in https://github.com/redhat-developer/odo/pull/1158/files#diff-08eb53b6a5d51aab5d95b32c1ebcead5R50

Reply:
According to my observation, `glob.Compile()` doesn't throw error when a wrong expression is passed. So this might be tricky to reproduce in test

Reply:
if that is true, than it might be a bug in the library. Why would it return an error?

Reply:
You can make it  fail with `[`

Reply:
Good, thanks :+1: